src/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -165,6 +165,14 @@ public static double Tanh(double value)
             return RuntimeImports.tanh(value);
         }
 
+        internal static double CopySign(double x, double y)
+        {
```

Comment:
It  may be better for the #if PLATFORM_UNIX ifdef to be moved here, so that this function is not unused for Windows, and can be potentially used from other places in future without ifdefs.